plugins/azure/automationAccounts/automationAcctApprovedCerts.js

```diff
@@ -0,0 +1,84 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Automation Account Approved Certificates Only',
+    category: 'Automation',
+    domain: 'Management and Governance',
+    description: 'Ensure that Azure Automation accounts are only using approved certificates.',
+    more_info: 'Certificates in azure automation accounts should be approved by the organization to meet the organizational security requirements. ',
```

Comment:
```suggestion
    more_info: 'Certificates in azure automation accounts should be approved by the organization to meet the organizational security requirements. ',
```
make it more descriptive, also it should be phrased that we don't use the word organization but mention that only approved certificates should be used